UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlInternals.cs

```diff
@@ -191,6 +191,8 @@ private void UpdateMe()
 
 		private void UpdateState()
 		{
+			if (PlayerManager.LocalPlayerScript.playerHealth.RespiratorySystem.CurrentBreathingTubes.Count > 0) isWearingMask = true;
```

Comment:
isn't this on the client so you have to network CurrentBreathingTubes?

Reply:
god dammit you right

Reply:
for this ill probably just network the count